python/lsst/daf/butler/registry/bridge/monolithic.py

```diff
@@ -237,7 +237,13 @@ def join_records(
         # Run query, transform results into a list of dicts that we can later
         # use to delete.
         with self._db.query(info_in_trash) as sql_result:
-            rows = [dict(**row, datastore_name=self.datastoreName) for row in sql_result.mappings()]
+            # can not use dict(**row, datastore_name=) because mypy knows that
```

Comment:
Can you do `dict(row, datastore=...)` instead?

Reply:
Very good. That does work.